src/index.html

```diff
@@ -4,9 +4,422 @@
+        </p>
+
+        <div class="card__rating">
+          <div class="stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+            Reviews: 5
+        </div>
+
+        <p class="card__price">
+          <span class="card__lable">Price:</span>
+
+          <span class="card__sum">$2,199</span>
+        </p>
+
+        <div class="card__button">
+          <a
+            href="#"
+            class="card__button-text"
+            data-qa="card-hover"
+          >
+          Buy
+          </a>
+        </div>
+      </article>
+
+      <article class="card">
+        <div class="card__img">
+          <img
+            class="card__item"
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27"
+          >
+        </div>
+
+        <p class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+
+        <p class="card__code">
+          Product code: 195434
+        </p>
+
+        <div class="card__rating">
+          <div class="stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+            Reviews: 5
+        </div>
+
+        <p class="card__price">
+          <span class="card__lable">Price:</span>
+
+          <span class="card__sum">$2,199</span>
+        </p>
+
+        <div class="card__button">
+          <a
+            href="#"
+            class="card__button-text"
+            data-qa="card-hover"
+          >
+          Buy
+          </a>
+        </div>
+      </article>
+
+      <article class="card">
+        <div class="card__img">
+          <img
+            class="card__item"
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27"
+          >
+        </div>
+
+        <p class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+
+        <p class="card__code">
+          Product code: 195434
+        </p>
+
+        <div class="card__rating">
+          <div class="stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+            Reviews: 5
+        </div>
+
+        <p class="card__price">
+          <span class="card__lable">Price:</span>
+
+          <span class="card__sum">$2,199</span>
+        </p>
+
+        <div class="card__button">
+          <a
+            href="#"
+            class="card__button-text"
+            data-qa="card-hover"
+          >
+          Buy
+          </a>
```

Comment:
```suggestion
          <a
            href="#"
            class="card__button-text"
            data-qa="card-hover"
          >
            Buy
          </a>
```

Check in all places please